src/components/Dropdown/SideBarDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useCallback } from "react";
+import Back from "../../assets/back.svg";
+import {
+  DropdownHeader,
+  DropdownItem,
+  DropdownList,
+  SideBarDropdown,
+  SideDropdownHeader,
+  SideDropdownItem,
+  SideDropdownsList,
+} from "./DropdownStyle";
+
+export interface DropdownProps {
+  title: string;
+  items: string[];
+  //selected: string;
+  onOptionClicked?: (item: string) => void;
+}
+export const SideDropdown = ({ title, items }: DropdownProps) => {
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const onOptionClicked = useCallback((value: string) => {
```

Comment:
You have to onOptionClick methods. if the one form the props is unused, delete it